### PR TITLE
README -> docs/README and generate main README + discord link

### PR DESCRIPTION
**Pull request by carlopi**
Not a very relevant AND not urgent contribution, but I had it this around already so I though of sending.

Adding link to discord + a rework of how README works so that once you are using the template you get a shorter and more to the point README, while keeping the other around.

### CODE REVIEW

**Comment by carlopi** [comment]
+ forgot to add the result of using the script: https://github.com/carlopi/extension-template-clone

File: docs/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # WIP Disclaimer
-This template is currently work-in-progress. Feel free to play around with it and give us feedback. Note also that this template depends on a development version of DuckDB. Follow https://duckdb.org/news for more information on official launch.
+This template is currently work-in-progress. Feel free to play around with it and give us feedback. Note also that this template depends on a development version of DuckDB.
```
Comment (Mause) [inline docs/README.md:2]:
```suggestion
This template is currently a work-in-progress. Feel free to play around with it and give us feedback. Note also that this template depends on a development version of DuckDB.
```

File: scripts/set_extension_name.py [hunk 2]
```diff
@@ -1,13 +1,15 @@
 #!/usr/bin/python3
 
-import sys, os
+import sys, os, shutil
 from pathlib import Path
 
-if (len(sys.argv) != 2):
-    raise Exception('usage: python3 set_extension_name.py <name_for_extension>')
+shutil.copyfile(f'docs/NEXT_README.md', f'README.md')
```
Comment (Mause) [inline scripts/set_extension_name.py:6]: Should this be a move?

**Review by Mause — CHANGES_REQUESTED** [review]

File: scripts/set_extension_name.py (hunk 2 above)
Comment (carlopi) [reply scripts/set_extension_name.py:6]: My idea was that copy makes for it to be doable again (say you want to change name a second time). Then some other stuff (like test file) will break, but I though was a first step in making it less state-dependent.

**Comment by samansmink** [comment]
looks good, thanks!

**Pull request merged**